src/test/java/uk/gov/dhsc/htbhf/smartstub/helper/PersonTestFactory.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.dhsc.htbhf.smartstub.helper;
+
+import uk.gov.dhsc.htbhf.smartstub.model.PersonDTO;
+
+import static uk.gov.dhsc.htbhf.smartstub.helper.AddressDTOTestDataFactory.aValidAddress;
+import static uk.gov.dhsc.htbhf.smartstub.helper.TestConstants.HOMER_DATE_OF_BIRTH;
+import static uk.gov.dhsc.htbhf.smartstub.helper.TestConstants.HOMER_FIRST_NAME;
+import static uk.gov.dhsc.htbhf.smartstub.helper.TestConstants.SIMPSON_LAST_NAME;
+
+/**
+ * Test data factory for {@link PersonDTO} objects.
+ */
+public class PersonTestFactory {
+
+    private static final String NINO = "EB123456C";
```

Comment:
I guess this is copy and paste again .. but is there a reason `NINO` is the only private constant?

Reply:
I've moved this to the TestConstants package.